stylometryproject/stylometryapp/static/scripts/dropdown.js

```diff
@@ -23,25 +23,25 @@ $(document).ready(function() {
                     'name': newProfileName,
                     // TO DO? - any new profiles fields in the future go here
                 },
-                success: function(data) {
+                success: function (data) {
                     // Create a new list item for the profile with the same structure and classes as the existing ones
                     var listItem = $('<li><a class="profile-item" href="#" data-profile-id="' + data.id + '">' + data.name + '</a></li>');
-                    
+
                     // Find the "New Profile" list item and insert the new item before it
-                    $('#profile-list li:last').before(listItem);
-                    
+                    $('#profile-list li:last').before(listItem);;
```

Comment:
I don't think the extra semi-colon should be here.

Reply:
you're right, ill fix now